lib/canvas.js

```diff
@@ -35,6 +36,9 @@ Canvas.prototype = {
     assert(Canvas.supportedFormats.indexOf(format) !== -1,
       '`gmsmith` doesn\'t support exporting a "' + format + '". Please export either a `png` or `jpeg`');
 
+    // Extent to given size
+    canvas.gravity('NorthWest').extent(this.width, this.height, '!');
```

Comment:
Instead of using extent/gravity, can we set width/height on `gmsmith.gm` itself?